libvirt/provider.go

```diff
@@ -8,7 +8,7 @@ import (
 func Provider() terraform.ResourceProvider {
 	return &schema.Provider{
 		Schema: map[string]*schema.Schema{
-			"uri": &schema.Schema{
+			"URI": &schema.Schema{
```

Comment:
Let's leave that to `uri` otherwise all the existing terraform files will be broken.